src/news/2018-01-31-uioPlus.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Introducing UI Options Plus (UIO+)
+filename: uioPlus
+date: '2018-01-31'
+---
+
+User Interface Options, also known as Learner Options, provides individuals with the ability
+to customize a web site to meet their personal needs and preferences. Currently, to provide such customizations,
+the web page needs to incorporate the UI Options widget into the site's markup.
+However, not every site will have UI Options or site personalizations incorporated.
+
+User Interface Options Plus (UIO+) aims to address this issue by enabling the personalization
+experience of UI Options as a browser extension.
+Preferences can be set via the UIO+ adjuster panel or, if on a
+[GPII (Global Public Inclusive Infrastructure)](https://gpii.net/) enabled machine, from a keyed in preference set.
+
+We have recently published a beta of [UIO+](https://chrome.google.com/webstore/detail/ui-options-plus-uio+/okenndailhmikjjfcnmolpaefecbpaek)
+to the Chrome Web Store.
+
+The following preferences are currently supported:
+
+- Text Size
+- Line Space
+- Contrast
+- Right-Click to Select
+- Selection Highlight
+- Text-to-Speech (requires the [click2speech](https://chrome.google.com/webstore/detail/click2speech/djfpbemmcokhlllnafdmomgecdlicfhj)
+  extension)
+- Reading Mode
+- Table of Contents
+- Enhance Inputs
+- Dictionary (requires the [Google dictionary](https://chrome.google.com/webstore/detail/google-dictionary-by-goog/mgijmajocgfcbeboacabfgobmjgjcoja)
+  extension)
+
+**Note**: The ability to apply an adaptation will vary from page to page.
+
+UIO+ is still early in development. We aim to improve/expand preference options and support
+ additional browsers in the future.
+
+UI Options Plus is the result of a joint effort of the [Inclusive Design Research Centre](https://idrc.ocadu.ca/) at
+[OCAD University](https://www.ocadu.ca/) and the [Trace R&D Center](https://trace.umd.edu/) at [University of Maryland](https://umd.edu/)
+under funding for the [FLOE Project](https://floeproject.org/) from the [William and Flora Hewlett Foundation](https://www.hewlett.org/)
+and the [National Institute on Disability, Independent Living, and Rehabilitation Research (NIDILRR)](https://www.acl.gov/about-acl/about-national-institute-disability-independent-living-and-rehabilitation-research)
+,[Administration for Community Living](https://www.acl.gov/) under grant #90RE5027.
```

Comment:
the comma at the beginning of this line should have a space after it, and not before it

Reply:
The space before the comma is still present, since it's on a new line

Reply:
fixed